Description: iperf does not handle right large time values
 Reported by Eugene Butan <eugene@mikrotik.com>
 When I invoke 'iperf' with '-t 100000000' argument from an ordinary shell
 prompt it immediately exits displaying incorrect bandwidth. If I supply
 smaller time value, iperf works as expected.
Author: Roberto Lumbreras <rover@debian.org>
Bug-Debian: http://bugs.debian.org/346099
Forwarded: https://sourceforge.net/tracker/index.php?func=detail&aid=3140391&group_id=128336&atid=711371

Index: iperf-2.0.5/src/Settings.cpp
===================================================================
--- iperf-2.0.5.orig/src/Settings.cpp	2012-05-24 19:16:22.000000000 +0200
+++ iperf-2.0.5/src/Settings.cpp	2014-07-04 10:59:48.369409924 +0200
@@ -458,7 +458,7 @@
         case 't': // seconds to write for
             // time mode (instead of amount mode)
             setModeTime( mExtSettings );
-            mExtSettings->mAmount = (int) (atof( optarg ) * 100.0);
+            mExtSettings->mAmount = (max_size_t) (atof( optarg ) * 100.0);
             break;
 
         case 'u': // UDP instead of TCP
